starfish/pipeline/features/pixels/pixel_spot_detector.py

```diff
@@ -9,19 +9,20 @@
 
 
 class PixelSpotDetector(PixelFinderAlgorithmBase):
-
     def __init__(
-            self, codebook: Codebook, distance_threshold: float,
-            magnitude_threshold: int, min_area: int, max_area: int, norm_order: int=2,
-            crop_x: int=0, crop_y: int=0, crop_z: int=0, **kwargs) -> None:
+            self, codebook: Codebook, metric: str, distance_threshold: float,
+            magnitude_threshold: int, min_area: int, max_area: int, norm_order: int = 2,
+            crop_x: int = 0, crop_y: int = 0, crop_z: int = 0, **kwargs) -> None:
```

Comment:
I could have sworn that they _required_ there to be no spaces around the equals operator, but then I went and found these examples and now I don't know what to do!

https://mypy.readthedocs.io/en/latest/cheat_sheet_py3.html?highlight=default#functions

@ttung

Reply:
this is the result of running Pycharm's "Reformat Code" tool, which adheres to whatever PEP-8y thing Pycharm defaults to. I can change the defaults if you'd like